app/scripts/models/email-resend.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This model limits the number of emails a component can send
+
+'use strict';
+
+define([
+  'backbone',
+  'lib/constants'
+], function (Backbone, Constants) {
+
+  function _shouldResend (tries) {
+    return tries === 1 || tries === 4;
+  }
+
+  var EmailResend = Backbone.Model.extend({
+    defaults: {
+      tries: 0,
+      maxTries: Constants.EMAIL_RESEND_MAX_TRIES,
+      onTriesExceeded: undefined,
```

Comment:
Two comments here:

1. We use `defaults` to only store the initial values of mutable "model" data that should be fetched when doing `toJSON` (basically, the actual list of data fields). Neither `onTriesExceeded` nor `maxTries` really fit this.
1. onTriesExceeded is a callback that is called once max tries has been exceeded. This isn't really needed since EmailResend is a Backbone Model. Backbone Models have an Events capability built in so you don't need to do the homegrown management, you should just be able to trigger a simple event - see http://backbonejs.org/#Events-trigger.